test/PowerShellEditorServices.Test/Extensions/ExtensionCommandTests.cs

```diff
@@ -48,34 +55,30 @@ public async Task CanRegisterAndInvokeCommandWithCmdletName()
                 BufferRange.None);
 
             EditorCommand commandAdded = null;
-            _extensionCommandService.CommandAdded += (object sender, EditorCommand command) =>
-            {
-                commandAdded = command;
-            };
-
-            string commandName = "test.function";
-            string commandDisplayName = "Function extension";
+            extensionCommandService.CommandAdded += (object _, EditorCommand command) => commandAdded = command;
 
+            const string commandName = "test.function";
+            const string commandDisplayName = "Function extension";
 
-            await _powershellContextService.ExecuteScriptStringAsync(
-                TestUtilities.NormalizeNewlines($@"
-function Invoke-Extension {{ $global:testValue = 5 }}
-Register-EditorCommand -Name {commandName} -DisplayName ""{commandDisplayName}"" -Function Invoke-Extension"));
+            await psesHost.ExecutePSCommandAsync(
+                new PSCommand().AddScript(
+                    "function Invoke-Extension { $global:extensionValue = 5 }; " +
+                    $"Register-EditorCommand -Name {commandName} -DisplayName \"{commandDisplayName}\" -Function Invoke-Extension"),
+                CancellationToken.None).ConfigureAwait(true);
```

Comment:
`ConfigureAwait(true)` is the default behavior, so why adding this call explicitly?
It looks you are emphasizing that current synchronization context needs to be captured, so where is the synchronization context come from?

Reply:
The code analyzers are setup to require `ConfigureAwait` to be set explicitly, because not setting `ConfigureAwait(false)` in the rest of the codebase (where it acts like a library) leads to a recurring bug when users load stuff like `Windows.Forms` into the extension's console.

The xUnit synchronization context is what's being captured, and in the tests it needs to be captured so the test code always returns to a thread that isn't the pipeline.

Reply:
> Consider calling ConfigureAwait on the awaited task [PowerShellEditorServices.Test] csharp(CA2007)

Reply:
https://github.com/PowerShell/PowerShellEditorServices/blob/70f31d9e5a0268a8e1512c735da675f342a08d94/.editorconfig#L31-L32

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2007 enabled in ec4bc2f44